sourcecode/weatherByYearMonthBar/weatherman.py

```diff
@@ -0,0 +1,19 @@
+from sourcecode.filehandling import FileHandling
+from sourcecode.report_generator import ReportGenerator
+
+class WeatherManBar(FileHandling):
```

Comment:
All printing related stuff could be placed in single class with all the functionalities.